internal/rbd/migration_test.go

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright 2021 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package rbd
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestIsMigrationVolHandle(t *testing.T) {
+	t.Parallel()
+	volID := "wrong_volume_handle"
+
+	if isMigrationVolHandle(volID) {
+		t.Error("request is not a migration delete request")
+	}
+
+	volID = "mig_mons-b7f67366bb43f32e07d8a261a7840da9_image-e0b45b52-7e09-47d3-8f1b-806995fa4412_pool_replica_pool" // nolint:lll // migration handle
+	if !isMigrationVolHandle(volID) {
+		t.Error("request is a migration delete request")
+	}
+
+	volID = "mig_image-e0b45b52-7e09-47d3-8f1b-806995fa4412_pool_replica_pool"
+	if !isMigrationVolHandle(volID) {
+		t.Error("request is a migration delete request")
+	}
+
+	volID = "mig_image-e0b45b52-7e09-47d3-8f1b-806995fa4412"
+	if !isMigrationVolHandle(volID) {
+		t.Error("request is a migration delete request")
+	}
+	volID = "mig_"
+	if isMigrationVolHandle(volID) {
+		t.Error("request is not a migration delete request")
+	}
+
+	volID = "kubernetes-dynamic-pvc-e0b45b52-7e09-47d3-8f1b-806995fa4412"
+	if isMigrationVolHandle(volID) {
+		t.Error("request is not a migration delete request")
+	}
```

Comment:
can this be rewritten with a simple for loop and a variable to check its migrationVolumeHandle or not?